docs/parameters.yaml

```diff
@@ -556,6 +556,20 @@ root_default: /var/cache/pelican/maxmind/GeoLite2-City.mmdb
 default: $ConfigBase/maxmind/GeoLite2-city.mmdb
 components: ["director"]
 ---
+name: Director.AdvertisementTTL
+description: >-
+  The time to live (TTL) of director's internal cache to store origins and caches advertisement.
+type: duration
+default: 15m
+components: ["director"]
+---
+name: Director.FileTransferInterval
```

Comment:
I think we should name this something like `Director.OriginHealthTestFreq`, or a name that highlights the service we're testing. While the description here is good, if someone comes across the config option in a config file without context, it might not be clear what this does.

Reply:
Agree. For upstream compatibility, let's name it `OriginCacheHealthTestInterval` as we are about to add test to caches.